SUMMARY.md

```diff
@@ -134,6 +134,7 @@
 		* [POST Update Call](voice/methods/calls/postCallsCallId.md)
 		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 	* [Conferences](voice/methods/conferences/about.md)
+		* [GET Account Conferences](voice/methods/conferences/getConferences.md)
```

Comment:
I might just say `GET Conferences` for this

Reply:
isn't it better to be consistent with the GET Recordings endpoint?

Reply:
Eh sure